src/ramses_tx/parsers.py

```diff
@@ -2129,6 +2129,24 @@ def parser_31d9(payload: str, msg: Message) -> dict[str, Any]:
     }
 
     if msg.len == 3:  # usu: I -->20: (no seq#)
+        if (
+            payload[:4] == "0000"
+            and msg._addrs[0] == msg._addrs[2]
+            and msg._addrs[1] == NON_DEV_ADDR
+        ):
+            # Vasco D60 HRU and ClimaRad minibox REM
+            from .ramses import _31D9_FAN_INFO
```

Comment:
This is unusual - please justify not doing the import at the top of teh module - add a comment, like: `# avoids circular import` if so